### DISCUSSION TITLE
need help for make AIIM

### DISCUSSION THREAD

**Discussion by stavealexe** [community/community#140958] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Body

AIIM sestem

I need help to make a privet AI network AIIM (Artificial Intelligence Identifier Module)
If you want to help me plz join AIIM sestem community : https://m.me/ch/AbZPSeyNaqNjLTG_/?send_source=cm%3Acopy_invite_link

**Comment by LiteBrite82** [comment]
Hi @stavealexe , we’re glad you’re here 😀

Thanks for sharing your project with the community. Unfortunately, we currently do not allow self-promotion or advertising on the Community Discussions. We want to make sure there is space for users to ask questions without overwhelming them with other conversations. Thank you for helping us maintain a productive and tidy community for all our members.

**Reply by stavealexe** [reply]
Oo.ok.
